src/features/comments/commentsSlice.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable no-param-reassign */
+import { createAsyncThunk, createSlice, PayloadAction } from '@reduxjs/toolkit';
+import * as commentsApi from '../../api/comments';
+import { Comment } from '../../types/Comment';
+
+export interface IPostState {
+  comments: Comment[],
+  loader: boolean,
+  error: boolean
+}
+
+const initialState: IPostState = {
+  comments: [],
+  loader: false,
+  error: false,
+};
+
+export const getCommentsAction = createAsyncThunk(
+  'comments/GET',
+  async (id:number) => {
+    const result = await commentsApi.getPostComments(id);
```

Comment:
try catch

Reply:
Why? if my editor handles all steps, including the error